Add example how to use mouse (#362)
AvaloniaUI/Documentation: docs/input/pointer.md

# Mouse & Pointer devices

> Avalonia operates on the abstraction called pointer devices. These devices including, but not limited to mouse, joystick, touchpad etc. Each control in Avalonia provides following events which allow developer subscribe to the pointer device movements, clicks and wheel movements: 

Reviewer: ```suggestion
Avalonia operates on the abstraction called pointer devices. These devices including, but not limited to mouse, touch, pen etc. Each control in Avalonia provides following events which allow developer to subscribe to the pointer device movements, clicks and wheel movements: 
```

Reviewer: We don't really support joystick now (unless it's emulated as a mouse), and "touchpad" is always emulated as a mouse on all platforms we support. Touch and Pen are unique types of pointer input though.

- PointerEnter
- PointerLeave
- PointerMoved
- PointerPressed
- PointerReleased
- PointerWheelChanged

> Example how to react on mouse button pressed.

Reviewer: ```suggestion
Example how to react on pointer button pressed.
```
while "mouse" is correct for 95% of use cases, I would force correct terminology here

```csharp
control.PointerPressed += (args) =>
{

>     var point = args.GetCurrentPoint(control);

Reviewer: ```suggestion
    var point = args.GetCurrentPoint();
```

see below

    var x = point.Position.X;
    var y = point.Position.Y;

>     if (point.Properties.IsLeftButtonPressed) {
>         // left button pressed
>     }
>     if (point.Properties.IsRightButtonPressed) {
>         // right button pressed
>     }

Reviewer: ```suggestion
    if (point.Properties.IsLeftButtonPressed)
    {
        // left button pressed
    }
    if (point.Properties.IsRightButtonPressed)
    {
        // right button pressed
    }
```

Reviewer: nit

}
```

> In the example above, coordinates `x` and `y` are relatively to control origin. If you want coordinates relative to your window, you may pass it to `PointerEventArgs.GetCurrentPoint` method like this: `var pointWindowCoords = args.GetCurrentPoint(window);`

Reviewer: ```suggestion
In the example above, coordinates `x` and `y` are relatively to the window origin. If you want coordinates relative to the specific control, you may pass it to `PointerEventArgs.GetCurrentPoint` method like this: `var pointControlCoords = args.GetCurrentPoint(control);`
```

Reviewer: I believe by default users want position relative to the root window, which is returned when you don't provide any control to this method.

> 

Reviewer: ```suggestion

# Tapped event args

Each control also has special gesture events: Tapped and DoubleTapped.
Tapped is raised, when pointer was pressed on the control and then released. 
While DoubleTapped is raised after pointer was pressed two times on the same place. Allowed size (distance between first and second "tap") and time (delay between them) depend on the platform and usually is bigger for touch devices. 

```
